html/david.html

```diff
@@ -29,7 +29,7 @@
               <ul class="nav">
                 <li class="active"><a href="index.html">Home</a></li>
                 <li><a href="index.html">Everything David</a></li>
-                <li><a href="index.html">David is Cool</a></li>
+                <li><a href="index.hml">David is Cool</a></li>
```

Comment:
huh. looks like I somehow had the broken version of your page on my machine. should be fixed now.